### PR TITLE
converted buttonlists to checkbox list

### PR DESCRIPTION
**Pull request by skumo**
hi @elisechant, have converted all btngroup lists to checkboxList ... this is how it looks like.
[form.pdf](https://github.com/CodeforAustralia/cese-snippet/files/1966070/form.pdf)

### CODE REVIEW

File: client/src/routes/account/components/programForm/form.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ class ProgramForm extends React.Component {
               <FormGroup row>
                 <Col md={8}>
                   <Label htmlFor="category">Program Focus Area</Label>
-                  <FieldRadioBtnList name="category"
+                  <FieldCheckboxList name="category"
```
Comment (elisechant) [inline client/src/routes/account/components/programForm/form.js]: should this not be FieldRadioList instead? different behaviour

File: client/src/routes/account/components/programForm/form.js [hunk 2]
```diff
@@ -288,6 +288,15 @@ class ProgramForm extends React.Component {
                                      value={values.yearLevels}
                                      options={this.optionsYearLevels}
                   />
+                  <input type="checkbox" className="form-check form-check-inline" onClick={(event) => {
+                    const selectAll = ['K', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'];
+                    event.target.checked
```
Comment (elisechant) [inline client/src/routes/account/components/programForm/form.js]: This is the right logic, but we can make it more Reacty

File: client/src/routes/account/components/programForm/form.js (hunk 1 above)
Comment (skumo) [reply client/src/routes/account/components/programForm/form.js]: oh ok .. get back to this ..

File: client/src/routes/account/components/programForm/form.js (hunk 2 above)
Comment (skumo) [reply client/src/routes/account/components/programForm/form.js]: @elisechant, now have a separate component ...

File: client/src/routes/account/components/programForm/form.js (hunk 1 above)
Comment (skumo) [reply client/src/routes/account/components/programForm/form.js]: @elisechant .. didnt realise these are radio buttons ...

**Comment by skumo** [comment]
@elisechant, pls review for the radioButton list changes ...

File: client/src/components/fieldRadioBtnList/index.js [hunk 3]
```diff
@@ -10,11 +9,19 @@ const FieldRadioBtnList = ({ options, name, value, onChange, onBlur, vertical =
           {options.map((o, idx) => {
             const isChecked = o.value === value;
             return (
-              <Button key={idx} color={isChecked ? 'secondary' : 'light' }
-                      onClick={() => {
-                        onChange(name, o.value);
-                      }}>
-                {o.label}</Button>
+              <div key={idx} color={isChecked ? 'secondary' : 'light' }>
```
Comment (elisechant) [inline client/src/components/fieldRadioBtnList/index.js]: I feel like this should be an additional component FieldRadioList instead of FieldRadioBtnList 

<img width="251" alt="screen shot 2018-05-05 at 8 00 30 am" src="https://user-images.githubusercontent.com/2195815/39654920-71fab6f4-503a-11e8-8400-5d40118ba817.png"

File: client/src/components/fieldRadioBtnList/index.js [hunk 4]
```diff
@@ -10,11 +9,19 @@ const FieldRadioBtnList = ({ options, name, value, onChange, onBlur, vertical =
...
+                <label >
```
Comment (elisechant) [inline client/src/components/fieldRadioBtnList/index.js]: ideally we should also be using Bootstrap classes to style here

File: client/src/routes/account/components/programForm/form.js [hunk 5]
```diff
@@ -288,6 +288,15 @@ class ProgramForm extends React.Component {
                                      value={values.yearLevels}
                                      options={this.optionsYearLevels}
                   />
+                  <input type="checkbox" className="form-check form-check-inline" onClick={(event) => {
```
Comment (elisechant) [inline client/src/routes/account/components/programForm/form.js]: This change belongs to https://github.com/CodeforAustralia/cese-snippet/pull/164 and shouldn't be part of the PR.

File: client/src/components/fieldRadioBtnList/index.js (hunk 3 above)
Comment (elisechant) [reply client/src/components/fieldRadioBtnList/index.js]: I've fixed this now at https://github.com/CodeforAustralia/cese-snippet/tree/master/client/src/components/fieldRadioList

**Comment by elisechant** [comment]
Closing because this has now been resolved in other PRs
